policyengine_us/parameters/gov/states/ga/tax/income/exemptions/dependent.yaml

```diff
@@ -0,0 +1,12 @@
+description: Georgia provides the following exemption amount for each dependent.
+values:
+  2003-01-01: 3_000  
+metadata:
+  period: year
+  unit: currency-USD
+  label: Georgia dependent exemption amount
+  reference:
+  - title: 2022 Form 500 Georgia Income Tax Return
```

Comment:
Add a page and line number reference. If possible could you find a different version of this that's searchable, e.g. a pdf? Same for the other reference.